server/shared/database/migrations/20190409125936-add-user-info.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Timestamp of the migration is before last one in develop ☝️

Reply:
Proper migration timestamp fixed by generating `add-user-info` migration again.